ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -164,9 +164,8 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                     }
                     .accessibilityLabel(attributedMessage.string)
             } else if let message = state.message {
-                let messageFont = hasSecondTextRow ? tokens.footnoteTextFont : (state.style.isToast ? tokens.boldTextFont : tokens.regularTextFont)
                 Text(message)
-                    .font(.fluent(messageFont))
+                    .font(.fluent(tokens.regularTextFont))
```

Comment:
Do we need to add a check to Title to make sure we aren't making it bold when there's only one line of text?

Reply:
This change takes care of that as it always sets the subtext to body 2.